server/public/model/post_test.go

```diff
@@ -995,3 +995,190 @@ func TestPostPriority(t *testing.T) {
 	p.Metadata.Priority.Priority = NewPointer(PostPriorityUrgent)
 	require.True(t, p.IsUrgent())
 }
+
+func TestPost_PropsIsValid(t *testing.T) {
+	tests := map[string]struct {
+		props   StringInterface
+		wantErr string
+	}{
+		"valid empty props": {
+			props:   nil,
+			wantErr: "",
+		},
+		"valid props": {
+			props: StringInterface{
+				"key": "value",
+			},
+			wantErr: "",
+		},
+		"invalid added_user_id type": {
```

Comment:
I miss the happy paths for all these validations.

Reply:
Good point, added via 891c599fc46865b9177f0957e2adae42bca07d03